javascript-bank/account.js

```diff
@@ -0,0 +1,42 @@
+/* exported Account */
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.deposit = function (amount) {
+  var deposit = new Transaction('deposit', amount);
+  if (deposit.amount > 0 && Number.isInteger(deposit.amount)) {
+    this.transactions.push(deposit);
+    return true;
+  } else {
+    return false;
+  }
+};
+
+Account.prototype.withdraw = function (amount) {
+  var withdrawal = new Transaction('withdrawal', amount);
+  if (withdrawal.amount > 0 && Number.isInteger(withdrawal.amount)) {
+    this.transactions.push(withdrawal);
+    return true;
+  } else {
+    return false;
+  }
+};
+
+Account.prototype.getBalance = function () {
+  var balance = 0;
+  if (this.transactions.length === 0) {
+    return balance;
```

Comment:
This conditional isn't necessary. If there is no length to the array, the loop would never start, so you're left with the same result.